commands/create_command.py

```diff
@@ -0,0 +1,30 @@
+from Icommand import ICommand
+import os, json, sys
+parent_dir = os.path.abspath(os.path.join(os.path.dirname(__file__), os.pardir))
+sys.path.append(parent_dir)
+from SchemaKeys import SchemaKeys
+
+class CreateCommand(ICommand):
+    def __init__(self, schema):
```

Comment:
parameter name should be a schema_path not a schema. naming variable is important.
Also I think I'm who confused you by this name. The input parameters in parser should be `schema_path`